Sources/Web3Core/Transaction/Envelope/LegacyEnvelope.swift

```diff
@@ -54,7 +54,7 @@ public struct LegacyEnvelope: AbstractEnvelope {
         toReturn += "Data: " + self.data.toHexString().addHexPrefix().lowercased() + "\n"
         toReturn += "Resolved chainID: " + String(describing: self.chainID) + "\n"
         toReturn += "- Intrinsic chainID: " + String(describing: self.explicitChainID) + "\n"
-        toReturn += "- Infered chainID: " + String(describing: self.impliedChainID) + "\n"
```

Comment:
Please undo this changes, as they affecting code, rather then comments and should be reviewed separately.